ONSdigital/response-operations-ui
# Js test ci validate

# Motivation and Context
Adds testing coverage to validate-ci functionality.  Involved refactoring functions to be more functional, and thus testable.

# What has changed
* Added test for validate-ci
* Refactored validate-ci for general improvement

# How to test?
* Ensure that selecting a CI on a CE shows no error if the filetype is `.xlsx`
* Use DevTools to remove `filetype` from the file input for CI.
* Ensure that selecting a non-`.xlsx` file shows an error
* Ensure uploading still works

# Links
https://trello.com/c/FI8fv3AE
